apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -40,6 +40,11 @@
     "SkipSignatureValidation": "SKIP_SIGNATURE_VALIDATION",
     "AppInsightsConnectionString": "APPLICATIONINSIGHTS_CONNECTION_STRING",
     "HardwarePublicTestKey": "HARDWARE_PUBLIC_TEST_PEM_BASE64_ENCODED",
-    "SlackStatusChannelWebhook": "SLACK_STATUS_CHANNEL_WEBHOOK"
+    "SlackStatusChannelWebhook": "SLACK_STATUS_CHANNEL_WEBHOOK",
+    "MailhogHost": "MAILHOG_HOST",
```

Comment:
What is Mailhog?

Reply:
I think you also need the Mailup server and the SMTP port

Reply:
Mailhog is a SMTP client used in a local environment for email testing. In my case (I work on a mac), I've installed mailohg via homebrew (alias brew) and I've configured a small SMTP server (postfix). This steps allows me to test the email sent with a specific template.

_(Only for more context, just for completeness)_
Since I've been really happy with mailhog (it's very convenient to use), I thought I'd allow other developers to configure things in two ways:
- Mailhog (in a local env, for testing): set only the `MailSender`, `MailhogHost` and `MailEnabled` variables to use mailhog tool and easily test email sending.
- MailUp (in a prod env): set only the MailSender, MailupUsername, MailupSecret and MailEnabled variables to use Mailhog and actually use the "bare and raw" MailUp service.

Reply:
The two things should not be mixed. If you are configuring MailUp this should be the MailUp configuration, if you want to use other services you should create an ad-hoc configuration for each service and use an adapter to change service. However, for a service in production I would rename MailhogHost to MailUpHost also for a question of compliance.